GiEnJul/ClientApp/src/common/functions/GetGender.tsx

```diff
@@ -0,0 +1,8 @@
+import { capitalize } from "@material-ui/core";
+import { GENDERS } from "../constants/Genders";
+
+export function getGender(key: number) {
+  const gender = GENDERS.find((element) => element.value == key)?.text;
+  if (gender == null) return "Udefinert";
+  return capitalize(gender);
+}
```

Comment:
This should probably changed to the code below. I think this would make it more usable. Also the constants/Genders.tsx variable should rely on this.
```suggestion
export function getGender(gender: Gender, age?: number) {
    if (age === undefined || age < 18) {
      switch (gender) {
        case Gender.Other:
          return "Ukjent";
        case Gender.Male:
          return "Gutt";
        case Gender.Female:
          return "Jente";
      }
    } else {
      switch (gender) {
        case Gender.Other:
          return "Ukjent";
        case Gender.Male:
          return "Mann";
        case Gender.Female:
          return "Kvinne";
      }
      return "Udefinert";
}
```

Reply:
Implemented in new commit